tasks/package.json

```diff
@@ -31,7 +31,6 @@
     "react-device-detect": "^2.2.3",
     "react-dom": "^18.2.0",
     "lucide-react": "^0.145.0",
-    "simplebar-react": "^3.2.3",
```

Comment:
I guess you never handled a merge conflict before.
That's not a problem.

Our current `main` branch needs this package in order to be operational.
As you added another package `lucide-react` to this project, your branch (`rajarshisamaddar:issue/174-replace-edit-with-edit-icon`) is not compatible with `helpwave:main` anymore cause another pull request demanding on `simplebar-react` got merged earlier.

To resolve this issue you normally just try to merge the `main` into this branch but you have to deal with conflicts manually. So just add both packages manually and do not delete the package introduced by these latest change in `main`.

Really good work, so far! 🚀 
We appreciate your cooperation very much and would be happy if you continue to help us develop our open source product (mozilla licensed). We have some minor bugs and issues with the label `low-prio`.

Reply:
Yeah, this is the first time with the merge conflict. I'll try to solve this conflict. @use-to I would like to be a part of this project.